corim-templates/README.md

```diff
@@ -0,0 +1,39 @@
+# How to ship Parsec TPM endorsements to Veraison
+
+* Install [`cocli`](https://github.com/veraison/corim/cocli/README.md):
+
+```shell
+go install github.com/veraison/corim/cocli@latest 
+```
+
+* Make a home for the CoMID intermediates:
```

Comment:
```suggestion
* Make a directory for the CoMID intermediates:
```

Reply:
done in [7b9a10f](https://github.com/CCC-Attestation/attested-tls-poc/pull/25/commits/7b9a10f52cf66632809e167cf6d68572a0b270e4)